recipes/atompaw/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+./configure --prefix=$PREFIX \
+   --with-linalg-libs="-L${PREFIX}/lib -lblas -llapack -lpthread"
```

Comment:
Can you enable shared?

Reply:
As far as I understand it is not supported by the developers https://github.com/atompaw/atompaw/blob/master/configure.ac#L173

Reply:
They are talking about inconvenience of not having a single executable, which is not something we have to worry about here.

Reply:
@isuruf I patched the configure script and updated the tests, can you take another look?